docs/spec-v1/concepts/perspectives.md

```diff
@@ -13,7 +13,7 @@ An application or service can be described both from a static or a dynamic persp
 
 ### Static Perspective
```

Comment:
I am missing the `system-type` related thought in ## ORD Aggregator Considerations, 
e.g. system-type is always added to instance specific, and also to system-version...